cloud/billing/account-management.md

```diff
@@ -1,37 +1,57 @@
 # Billing - Account management
 If you haven't yet added a payment method, you can add one in this section. You
-can also update to a new credit card here.
+can also update to a new credit card here. 
 
+## Add, update, or delete a payment method
+You can add up to three credit cards to the `Wallet` and set one of the cards as
+`primary` for payments. If you want to delete the last card in the `Wallet`, you
+have to first delete all the services and then schedule the last card to be
+deleted at the end of the month.
 <procedure>
 
-### Adding or changing a payment method
+### Adding, updating, or deleting a payment method
 1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
     the `Billing` details section.
-1.  Click `Add a payment method`. If you already have a payment method saved,
-    you need to delete it before you start. Click the trash can icon to delete
-    your saved credit card.
-1.  Type your credit card details, and click `Add billing address`.
-1.  Type your address, and click `Save payment method`.
-1.  Confirm your new credit card is saved by checking the details on
-    the `Billing` page.
+1.  In the `Wallet` section, click `Add a payment method`. If you already have
+    three credit cards saved in the payment methods, you need to delete at least
+    one of them before you start. Click the trash can icon to delete your saved
+    credit card.
+1.  Type your credit card details, and click `Continue`.
+    The `Add payment method: billing address` page appears.
+1.  Type you billing address and company address. The company address is
+    required to determine any applicable sales tax.
+    * If your billing address is the same as your company address, select
+      `Billing  address is the same as company address`, and click `Save`.
+    * If your billing address is not the same as your company address, click
+      `Continue`. Type your company address, and click `Save`.        
+1.  Confirm your new credit card is saved by checking the details on the
+    `Billing` page. To update your billing address and company address, click
+    the edit icon in the saved payment method.
```

Comment:
Not sure it is worth mentioning, but currently editing the company address is only possible with the primary CC. This is a bit confusing, but since we don't store the company address per card, we wanted to have only one place where the Company address can be changed. In the future, we plan to move the company address to a separate page anyway.

Reply:
Thank you for reviewing @aprimakina . Incorporated the changes you suggested in the latest commit.